README.md

```diff
@@ -1,14 +1,78 @@
-# Klarna Checkout SDK for mobile apps
+# Klarna Checkout SDK for Mobile Apps
+[![CocoaPods](https://img.shields.io/badge/cocoapods-v1.6.6-blue.svg?style=flat)](https://cocoapods.org/pods/KlarnaCheckoutSDK)
+[![Maven](https://img.shields.io/maven-metadata/v/https/x.klarnacdn.net/mobile-sdk/com/klarna/checkout/sdk/maven-metadata.xml.svg)](https://github.com/klarna/kco-mobile-sdk/blob/master/android/README.md)
+[![Carthage compatible](https://img.shields.io/badge/carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![License](https://img.shields.io/github/license/klarna/kco-mobile-sdk)](https://github.com/klarna/kco-mobile-sdk/blob/master/LICENSE)
+![Platforms](https://img.shields.io/badge/platform-ios%20%7C%20android-lightgrey)
 
-The SDK allows to integrate [Klarna Checkout](https://www.klarna.com/us/business/sell-online-with-klarna) in 
-native or hybrid iOS and Android applications. 
+The Checkout SDK allows to integrate [Klarna Checkout](https://www.klarna.com/us/business/sell-online-with-klarna) in native or hybrid iOS and Android applications.
 It's aimed to make integration simple and provide the smoothest in-app purchase experience.
 
+**Looking for Klarna Payments?** Check out the [Klarna In-App SDK](https://github.com/klarna/klarna-mobile-sdk) repo instead.
+
+## Requirements
+* iOS 10 or later.
+* Android 4.0 or later.
+
+## Support
+If you are having any issues using the SDK in your project, please create a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/klarna-inapp-sdk) tagged with `klarna-inapp-sdk`. If you think that something is wrong with the SDK itself, please create an issue.
```

Comment:
The Stackoverflow tag here is the same as In-App SDK. Is it ok?

Reply:
Yes, all questions about Klarna products will be under the same tag.